documentation-website/index.html

```diff
@@ -4,8 +4,8 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="./src/assets/favicon.svg"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta name="description" content="A documentation site for the open-source benchmarking framework @idrinth/api-bench">
-    <title>@idrinth/api-bench</title>
+    <meta name="description" content="A documentation site for the open-source benchmarking framework @idrinth-api-bench/api-bench">
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted

Reply:
Reverted